Resources/Prototypes/Entities/Structures/Machines/chem_master.yml

```diff
@@ -61,15 +60,20 @@
   - type: ContainerContainer
     containers:
       beakerSlot: !type:ContainerSlot
+      outputSlot: !type:ContainerSlot
       machine_board: !type:Container
       machine_parts: !type:Container
   - type: ItemSlots
     slots:
       beakerSlot:
-        whitelistFailPopup: chem-master-component-cannot-put-entity-message
         whitelist:
           components:
           - FitsInDispenser
+      outputSlot:
+        whitelistFailPopup: chem-master-component-cannot-put-entity-message
+        whitelist:
+          components:
+          - FitsInChemMasterOutput
```

Comment:
this could just be a tag instead, since the component itself seems to just be exactly that and doesn't seem to have any other use other than signifying that this entity can't fit in the chemmaster (see `Content.Shared/Whitelist/EntityWhitelist.cs` for more information, specifically the Tags field, and `Content.Shared/Tag/TagComponent.cs` for where you would store your tags)